docs/deprecated.md

```diff
@@ -96,7 +96,7 @@ on all subcommands (due to it conflicting with, e.g. `-h` / `--hostname` on
 ### `-e` and `--email` flags on `docker login`
 **Deprecated In Release: [v1.11.0](https://github.com/docker/docker/releases/tag/v1.11.0)**
 
-**Removed In Release: [v1.14.0](https://github.com/docker/docker/releases/)**
```

Comment:
Since the feature is already removed in the master branch, maybe it is OK to keep this line, but I also understand a reader may get confused because v1.14.0 is not released yet.

So, my suggestion is:
```
**Target For Removal In Release: v1.14 (already removed in the master branch)**
```

PTAL @thaJeztah